### PR TITLE
fix(prometheus_bucket_size): change prometheus bucket size from strin…

### PR DESCRIPTION
**Pull request by anusha5695**
Have updated the prometheus bucket_size from string to array
Added test for the same. (have also grouped together other prometheus tests under a single describe context)

| Question                | Answer                                                                          |
| ----------------    | -------------------------------------------------------------------------- |
| Bug fix                    | ✔                                                                        |
| New feature           | ✖                                                                        |
| Breaking change   | ✖                                                                        |
| Deprecations         | ✖                                                                        |
| Documentation      | ✔                                                                |
| Tests added           | ✔                                                            |

Related issues: https://github.com/Zooz/predator/issues/505

### CODE REVIEW

File: tests/integration-tests/configManager/configHandler-test.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ const updateBodyWithTypes = {
     },
     prometheus_metrics: {
         push_gateway_url: 'string_value',
-        buckets_sizes: 'string_value',
+        buckets_sizes: [2.1, 1, 10, 17, 19, 0.0],
```
Comment (enudler) [inline tests/integration-tests/configManager/configHandler-test.js:37]: I suggest change the examples to:
[0.01, 0.05, 0.10, 0.25, 0.50, 0.100, 0.200, 0.300, 0.400, 0.500, 1, 2, 5, 10, 30, 60, 120]

File: tests/integration-tests/configManager/configHandler-test.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ const requestBody = {
     },
     prometheus_metrics: {
         push_gateway_url: 'string_value_push_gateway_url',
-        buckets_sizes: 'string_value_buckets_sizes',
+        buckets_sizes: [9, 11, 0.1, 12, 78, 1.9],
```
Comment (enudler) [inline tests/integration-tests/configManager/configHandler-test.js:79]: same here:
[0.01, 0.05, 0.10, 0.25, 0.50, 0.100, 0.200, 0.300, 0.400, 0.500, 1, 2, 5, 10, 30, 60, 120]

**Comment by enudler** [comment]
Thank you @anusha5695 for this PR
Left small comment

File: tests/integration-tests/configManager/configHandler-test.js (hunk 1 above)
Comment (anusha5695) [reply tests/integration-tests/configManager/configHandler-test.js]: Sure, changed now.

File: tests/integration-tests/configManager/configHandler-test.js (hunk 2 above)
Comment (anusha5695) [reply tests/integration-tests/configManager/configHandler-test.js]: Done

**Comment by anusha5695** [comment]
@enudler Thank you. Have changed the array values :)

**Pull request merged**